### PR TITLE
The history does not store the reconciled entries

### PR DESCRIPTION
**Pull request by guewen**
Because the new reconciliations are committed in a different transaction
than the one used to create the history.
Move the new cursor initialization to the outer method 'run_reconcile()'
so everything is enclosed in the same transaction.

### CODE REVIEW

File: account_easy_reconcile/__init__.py [hunk 1]
```diff
@@ -23,3 +23,4 @@
 import base_reconciliation
 import simple_reconciliation
 import easy_reconcile_history
+import res_config  # noqa
```
Comment (bwrsandman) [inline account_easy_reconcile/__init__.py]: please remove the `# noqa`

**Comment by yvaucher** [comment]
@bwrsandman fixed noqa

**Comment by bwrsandman** [comment]
:+1:

**Comment by guewen** [comment]
Rebased

**Comment by pedrobaeza** [comment]
A little PEP8 issue:

```
./account_easy_reconcile/easy_reconcile.py:268:21: E124 closing bracket does not match visual indentation
```

**Comment by guewen** [comment]
Yes, right. Thanks

**Comment by pedrobaeza** [comment]
:+1:

**Comment by lepistone** [comment]
:+1: thanks

**Comment by lepistone** [comment]
Note: I relaunched the last build and it fails because we changed the line length from 80 to 79 characters. This is unrelated, so I merge this one and make a new PR to fix the line length on the repo.

**Pull request merged**